data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/AuditMessageService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018-2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.data.service;
+
+import akka.stream.Materializer;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+
+import uk.gov.gchq.palisade.service.data.model.TokenMessagePair;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Service for sending messages to the audit-service. Messages can be either success or error message type.
+ */
+public class AuditMessageService {
```

Comment:
This might be better named (see filtered-resource-service) as AuditEventService
```suggestion
public class AuditEventService {
```

Reply:
I would like to stick with AuditMessageService.   The idea of message gives to more of a sense that this service will do something with information.  Events for me has too strong of an association with the things that happen when a task is completed , like a mouse has been clicked.